test/src/core/stdlib.js

```diff
@@ -295,6 +295,7 @@ describe('foam.Function', function() {
     expect(foam.Function.functionComment(function() { })).toEqual('');
     expect(foam.Function.functionComment(function() {/**/ })).toEqual('');
     expect(foam.Function.functionComment(function() {/* hello */ })).toEqual('hello ');
+    expect(foam.Function.functionComment(function() {` hello ` })).toEqual('');
```

Comment:
It is not what I expected.

Reply:
Yes, this doesn't look correct. This should work like on line 297.

Reply:
please remove this line